### PR TITLE
Changed 'confirm password' placeholder text on user registration form

### PR DESCRIPTION
**Pull request by kumarnitj**
before:-
password

after:-
confirm password

### CODE REVIEW

File: packages/rocketchat-ui-login/client/login/form.html [hunk 1]
```diff
@@ -85,7 +85,7 @@ <h2 data-i18n="Registration_Succeeded">{{{_ "Registration_Succeeded"}}}</h2>
 									<div class="rc-input__wrapper">
 										<input name="confirm-pass" id="confirm-pass" type="password" class="rc-input__element"
 											autocapitalize="off" autocorrect="off"
-											placeholder="{{passwordPlaceholder}}" autofocus>
+											placeholder="Confirm Password" autofocus>
```
Comment (graywolf336) [inline packages/rocketchat-ui-login/client/login/form.html]: Please do not remove any i18n and instead add an additional place. And please remove the `package-lock.json` file changes.

Comment (kumarnitj) [reply packages/rocketchat-ui-login/client/login/form.html]: sorry, but I'm new in open Source. Could you please describe what do you mean by add an additional place and is my approach correct

Comment (kaiiiiiiiii) [reply packages/rocketchat-ui-login/client/login/form.html]: Replace “Confirm Password” with a placeholder and add a translation to the i18n file.

Comment (kumarnitj) [reply packages/rocketchat-ui-login/client/login/form.html]: should I add the same for confirm password  here in the form.js file in same folder. on the other hand there are many i18n files which one to use

Comment (kaiiiiiiiii) [reply packages/rocketchat-ui-login/client/login/form.html]: Forget my last comment.

There's a lot more to do, as `passwordPlaceholder` is not an i18n placeholder, it's a thru the admin ui configurable variable. Best thing to do would be an extra `confirmPasswordPlaceholder` option, configurable thru the admin ui. In addition you could add a default value (i18n translated) which will be shown, when there's no placeholder text set.

Search for "passwordPlaceholder" in your editor to get a good start:
screen shot 2018-03-02 at 19 32 09

**Comment by kumarnitj** [comment]
please review this pull request

File: packages/rocketchat-ui-login/client/login/form.html (hunk 1 above)
Comment (kumarnitj) [inline packages/rocketchat-ui-login/client/login/form.html]: I have done that i think, please review it

**Comment by kumarnitj** [comment]
please consider these changes

**Comment by ggazzo** [comment]
hey @kumarnitj thanks for your contribution, I made some changes in your PR, I thought you wouldn't matter about that.

**Pull request merged**